free/src/main/scala/cats/free/Coyoneda.scala

```diff
@@ -17,29 +18,32 @@ sealed abstract class Coyoneda[F[_], A] extends Serializable { self =>
   /** The underlying value. */
   val fi: F[Pivot]
 
-  /** The transformer function, to be lifted into `F` by `run`. */
-  val k: Pivot => A
+  /** The transformer functions, to be lifted into `F` by `run`. */
+  private[cats] val k: List[Any => Any]
 
-  import Coyoneda.{Aux, apply}
+  def execute(a: Pivot): A =
```

Comment:
Good question: it should in my opinion be public because a) it's as safe as exposing `k` previously and b) `k` was public before, which means users have had time to create use-cases that require using the function inside a `Coyo` on other values.